.travis.py

```diff
@@ -1,8 +1,35 @@
 import json
+import operator
 import os
 import re
 import subprocess
 
+from jsonschema import Draft4Validator
```

Comment:
need to create `requirements.txt` and add the package to it

Reply:
thanks for reviewing. Corrected now.